emission/integrationTests/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# python 3
+FROM ubuntu:latest
+
+RUN apt-get update
+RUN apt-get install -y curl wget
```

Comment:
nit: why do we need both `curl` and `wget`?

Reply:
`wget` is used in the run command to donwload the .tar.gz file for dockerize. `curl` is used in setup_conda.sh: `curl -o miniconda.sh -L https://repo.continuum.io/miniconda/Miniconda3-py39_$EXP_CONDA_VER-1-$PLATFORM.sh;` 

I tested out replacing `curl` with `wget`, and it does not work

Reply:
Did you try replacing wget with curl? Also "did not work" doesn't really mean much - **how** did it not work?
This may be moot anyway given that you don't need dockerize, in which case you should remove `wget`

Reply:
Dockerize removal committed